exercises/space-age/space_age_test.rb

```diff
@@ -3,28 +3,24 @@
 require 'minitest/autorun'
 require_relative 'space_age'
 
+# Common test data version: 7c63e40
 class SpaceAgeTest < Minitest::Test
   DELTA = 0.01
 
-  def test_age_in_seconds
-    age = SpaceAge.new(1_000_000)
-    assert_in_delta 1_000_000, age.seconds, DELTA
-  end
-
-  def test_age_in_earth_years
-    skip
+  def test_age_on_earth
+    # skip
     age = SpaceAge.new(1_000_000_000)
     assert_in_delta 31.69, age.on_earth, DELTA
   end
 
-  def test_age_in_mercury_years
+  def test_age_on_mercury
     skip
     age = SpaceAge.new(2_134_835_688)
     assert_in_delta 67.65, age.on_earth, DELTA
```

Comment:
I don't like that the `on_earth` method is also being tested here, I wouldn't do it in real life code, but I cannot tell if this test has educational value and should remain. 

https://github.com/exercism/xruby#workload-philosophy

Thoughts?